source/user/eph.rst

```diff
@@ -0,0 +1,97 @@
+.. _user_eph:
+
+**********************
+Electron Phonon Matrix
+**********************
+
+*Modules*: :mod:`eph`, :mod:`pbc.eph`
+
+Introduction
+============
+Electron-Phonon (eph) Interaction describes the coupling between the electrons and the nuclear degree of freedom. First order eph matrix elements are most commonly used to compute electron/phonon transport properties. PySCF supports first order eph matrix calculation by either Coupled Perturbed Kohn Sham/Hartree Fock (CPKS/CPHF) or finite difference. In periodic system, only Gamma-point calculation with FFTDF is supported.
+
+A minimal example of analytical eph matrix calculation is as follows::
+
+    from pyscf import gto, scf, eph
+    mol = gto.M(
+    atom = [['O', [0.000000000000,  -0.000000000775,   0.923671924285]],
+            ['H', [-0.00000000000,  -1.432564848017,   2.125164039823]],
+            ['H', [0.000000000000,   1.432564848792,   2.125164035930]]],
+    basis = 'sto3g',
+    unit = 'Bohr'
+    )
+    # Note that this is a pre-computed relaxed molecule
```

Comment:
So the method is for molecules? I have always associated phonons with the solid state.

If the geometry needs to be relaxed, as for vibrational analysis, one should note this also in the main text

Reply:
The pyscf.eph module itself is meant for molecules and pbc.eph is meant for crystals. And right, the concept of phonon originated from the solid state community and is only meant for lattice dynamics. I don't know if the quantum chemistry community has a similar jargon for the second quantized coupling Hamiltonian. Perhaps we need a different name for the module @gkc1000

Reply:
One can use it for molecules, although it is not widely used. It is basically making the harmonic approximation when computing non-adiabatic coupling. 

@yangcal I agree that we should state that this needs to be done at the equilibrium geometry to have any physical meaning.

Reply:
sure, I'll add more intro on this part